Gitlab/two-stage-pipeline-template/{{cookiecutter.outputDir}}/.gitlab-ci.yml

```diff
@@ -0,0 +1,170 @@
+variables:
+  SAM_TEMPLATE: {{cookiecutter.sam_template}}
+  PIPELINE_USER_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
+  PIPELINE_USER_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
+  TESTING_STACK_NAME: {{cookiecutter.testing_stack_name}}
+  TESTING_REGION: {{cookiecutter.testing_region}}
+  TESTING_PIPELINE_EXECUTION_ROLE: {{cookiecutter.testing_pipeline_execution_role}}
+  TESTING_CLOUDFORMATION_EXECUTION_ROLE: {{cookiecutter.testing_cloudformation_execution_role}}
+  TESTING_ARTIFACTS_BUCKET: {{cookiecutter.testing_artifacts_bucket}}
+  {%- if cookiecutter.testing_ecr_repo %}
```

Comment:
want to make sure the `-` sign positioning inside `{% %}` match the one in Jenkins template. If we missplace some of the signs, we might end up with missing new lines or extra new lines

Reply:
Sure!